mozilla/fx-private-relay
# for MPP-2174: add text stats

This PR fixes half (i.e., the "texts" stats) of MPP-2173.
![image](https://user-images.githubusercontent.com/71928/189179133-81b3f7ed-2397-48a3-9cdd-849daff387da.png)

How to test:
1. Make sure you have set up [local end-to-end phone dev](https://github.com/mozilla/fx-private-relay/blob/main/docs/end-to-end-local-phone-dev.md).
2. Run `python manage.py migrate` to add the new DB fields for stats
3. Go to http://127.0.0.1:8000/phone/
   * [x] Verify the stats show 50 minutes and 75 texts remaining
5. Send a text message to your relay number
   * [x] Verify the stats show 74 texts remaining, 1 calls and texts forwarded
6. Toggle your relay number to blocking
7. Send a text message to your relay number
   * [x] Verify the stats show 74 texts remaining, 1 calls and texts forwarded, 1 calls and texts forwarded

- [x] TODO: I've added a unit test to test for potential regressions of this bug.
